src/hooks/series/useSeries.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery, UseQueryResult } from 'react-query';
+
+import { getEpisodes, getMediaById, getSeries } from '#src/services/api.service';
+import type { Series } from '#types/series';
+import type { Playlist, PlaylistItem } from '#types/playlist';
+import type { ApiError } from '#src/utils/api';
+
+// Series and media items have the same id when creating via dashboard using new flow
+export default (seriesId: string | undefined): UseQueryResult<{ series: Series; media: PlaylistItem; playlist: Playlist }, ApiError> => {
+  return useQuery(
+    ['series', seriesId],
+    async () => {
+      const [series, media, episodesData] = await Promise.all([getSeries(seriesId || ''), getMediaById(seriesId || ''), getEpisodes(seriesId || '', 0)]);
```

Comment:
I kind of think that we should just get the series and the media item here and not couple getting the episodes into this. They can be retrieved for specific page contexts more efficiently (i.e. here we don't have the desired filters and such.)

Reply:
Looks like I don't need this data anymore. Removed this part.